### DISCUSSION TITLE
메인페이지의 Tab을 router로 페이지 변경으로 바꾸는건 어떨까?

### DISCUSSION THREAD

**Discussion by hy57in** [teamsindy20/sobok#48] [Design]
antd의 tab은 커스텀하기가 어렵다.
메인페이지의 경우 header와 함께 tab의 상단 메뉴들도 fixed가 되어야 하는데 antd의 경우 그 디자인을 적용할 수 없다
그래서 페이지 변경으로 바꾸는게 어떨까?

**Comment by hy57in** [comment]
아 구분되어있군요.
그럼 tab ㅅ타일..

**Reply by gwak2837** [reply]
탭 스타일 관련해선 더 찾아봐야 할 거 같아요
